src/diffusers/models/controlnet.py

```diff
@@ -517,7 +517,7 @@ def forward(
 
         controlnet_cond = self.controlnet_cond_embedding(controlnet_cond)
 
-        sample += controlnet_cond
+        sample = sample + controlnet_cond
```

Comment:
How does this break compilation? I don't get it 🤯

Reply:
I don't think this is necessary for `torch.compile` on nightly. I'm guessing on an older release it was just breaking for no good reason :)

Reply:
Thanks for the clarification!